### PR TITLE
Add: responsiveness for mobile and tablet views

### PR DESCRIPTION
**Pull request by shadabkhan16**
## Description
Responsiveness for mobile and tablet views in the airdrop component.

- What does this PR do?
- Why is it needed?

## Screenshots
Include screenshots for **both desktop and mobile** versions.

### Desktop:
[Add screenshot here]
Screenshot 2024-12-02 at 5 12 14 PM

### Mobile:
[Add screenshot here]

## Approach
Use breakpoints  

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by shadabkhan16** [comment]
Astronaut image added in the same branch

File: src/game_frontend/src/components/dashboard/airdrop/Tab/index.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 .tokenpanel,
-.tokenpanel-body > tr{
-    @apply w-full;
+.tokenpanel-body > tr {
+  @apply w-full lg:w-[35vw];
 }
-.tokenpanel-body > tr{
-    @apply flex justify-between items-center gap-4 font-coin text-2xl;
+.tokenpanel-body > tr {
+  @apply flex justify-between items-center gap-2 font-coin text-2xl pt-4;
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/airdrop/Tab/index.css:7]: Mobile screens should have smaller `text-size`. Look into shorter text sizes

File: src/game_frontend/src/pages/Dashboard/Airdrop/index.css [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
-    @apply self-stretch bg-white w-0.5;
+
+.airdrop-body {
+  @apply grid  grid-cols-1 lg:grid-cols-2 gap-4  relative;
+}
+.spacer {
+  @apply self-stretch bg-white w-0.5;
+}
+.page-title {
+  @apply font-bold text-5xl;
```
Comment (ArjunQBTech) [inline src/game_frontend/src/pages/Dashboard/Airdrop/index.css:16]: Smaller text-size for mobile screen

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]
Look into the suggested changes

**Review by ArjunQBTech — APPROVED** [review]
Looks good

**Pull request merged**